<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>车牌识别</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBoxPOWER">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxCAM">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string> 功率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>67</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>摄像头</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>  分辨率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>  帧率</string>
    </property>
   </widget>
   <widget class="QPushButton" name="beginButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>结束</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxPX">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxFPS">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableView" name="tableViewResults">
    <property name="geometry">
     <rect>
      <x>685</x>
      <y>121</y>
      <width>391</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditThreshold">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>      阈值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>   风扇</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxFAN">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1102</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
